File: python/two-fer/two_fer.py
'''Two-fer or 2-fer is short for two for one.

One for you and one for me.

Given a name, return a string with the message:

One for X, one for me.
Where X is the given name.

However, if the name is missing, return the string:

One for you, one for me.
'''


def two_fer(name: str = "you") -> str:
    '''Returns a string "One for {name}, one for me."'''
    return f"One for {name}, one for me."

File: javascript/two-fer/two-fer.js
/**
 * Two-fer or 2-fer is short for two for one. One for you and one for me.
 * 
 * Given a name, return a string with the message:
 * 
 * <p>One for name, one for me.
 * <p>Where "name" is the given name.
 * 
 * However, if the name is missing, return the string:
 * 
 * <p>One for you, one for me.
 * @file
 * @author Andreas Atle, atle.andreas@gmail.com
 */

/**
 * Returns a string 'One for {name}, one for me'.
 * @func
 * @param {string} [name='you'] The name of the person.
 * @returns {string} the wanted string.
 */
export const twoFer = (name = "you") => {
  return "One for " + name + ", one for me.";
};
